### PR TITLE
chore: update CLI installation instructions for Windows

### PR DESCRIPTION
**Pull request by khorne3**

### CODE REVIEW

File: cli/installation.md [hunk 1]
```diff
@@ -23,15 +23,25 @@ version number.
...
    `PATH` environment variable
 
+### Download (Windows)
+
+1. Click a release (e.g., `coder-cli-windows.zip`) and download the file
+1. Unzip the file
+1. Copy the file to `%USERPROFILE%\bin`
+1. On the command-line interface, execute `coder.exe login`
+
+> Make sure that you have a user variable `PATH` that's set to
```
Comment (johnstcn) [inline cli/installation.md:42]:
```suggestion
> Make sure that your user's `PATH` environment variable 
> contains the location you chose.
```

File: cli/installation.md [hunk 2]
```diff
@@ -23,15 +23,25 @@ version number.
...
-   coder-cli-linux-amd64.tar.gz)
+   `coder-cli-linux-amd64.tar.gz`)
 1. Extract the `coder` binary and copy it to a location you've added to your
    `PATH` environment variable
 
+### Download (Windows)
+
+1. Click a release (e.g., `coder-cli-windows.zip`) and download the file
+1. Unzip the file
+1. Copy the file to `%USERPROFILE%\bin`
```
Comment (johnstcn) [inline cli/installation.md:39]:
```suggestion
1. Copy the file to a location of your choosing (for example, `%USERPROFILE%\bin`).
```

**Review by johnstcn — APPROVED** [review]
Thanks for adding this! I don't have a Windows system easily at hand to verify these unfortunately.

**Comment by khorne3** [comment]
I also don't have a Windows system -- @BrunoQuaresma would you be able to help?

**Pull request merged**